animation_nodes/nodes/bvh_tree/is_inside_volume.py

```diff
@@ -30,24 +23,4 @@ def getExecutionCode(self, required):
         return "isInside = self.execute_Single(bvhTree, vector)"
 
     def execute_Single(self, bvhTree, vector):
-        hits1 = self.countHits(bvhTree, vector, direction1)
-        if hits1 == 0: return False
-        if hits1 == 1: return True
-
-        hits2 = self.countHits(bvhTree, vector, direction2)
-        if hits1 % 2 == hits2 % 2:
-            return hits1 % 2 == 1
-
-        hits3 = self.countHits(bvhTree, vector, direction3)
-        return hits3 % 2 == 1
-
-    def countHits(self, bvhTree, start, direction):
-        hits = 0
-        offset = direction * 0.0001
-        location = bvhTree.ray_cast(start, direction)[0]
-
-        while location is not None:
-            hits += 1
-            location = bvhTree.ray_cast(location + offset, direction)[0]
-
-        return hits
+        return isInsideVolume(bvhTree, vector)
```

Comment:
Call the function inside the execution code directly.

Reply:
If I write like this 
```
def getExecutionCode(self, required):
        return "isInside = isInsideVolume(bvhTree, vector)"
```
then it gives an error `NameError: name 'isInsideVolume' is not defined`.

Should I write normal `execute(self, )` function?

Reply:
It should be called from the AN module `isInside = AN.utils.bvh.isInsideVolume(bvhTree, vector)`.

Reply:
Okay, Thanks.